modules/standard/Set.chpl

```diff
@@ -251,26 +268,16 @@ module Set {
       var result = true;
 
       on this {
-        _enter();
+        _enter(); defer _leave();
 
+        // TODO: Acquire locks while iterating?
```

Comment:
Is there an issue about this? I'm worried that it won't be obvious to users that `isDisjoint` is iterating over the container.

Reply:
That is why the comment is there, I just shortened it from the previous mess that I wrote.